curations/go/golang/golang.org/x/arch.yaml

```
@@ -13,6 +13,9 @@ revisions:
   v0.4.0:
     licensed:
       declared: BSD-3-Clause AND LicenseRef-scancode-google-patent-license-golang
+  v0.6.0:
+    licensed:
+      declared: BSD-3-Clause
```

Now that I see the context, looks like:

```suggestion
      declared: BSD-3-Clause AND LicenseRef-scancode-google-patent-license-golang
```